cbw_api_toolbox/cbw_api.py

```diff
@@ -67,11 +74,15 @@ def ping(self):
 
     def servers(self):
         """GET request to /api/v2/servers to get all servers"""
-        return [CBWParser().parse(CBWServer, server) for server in self._request(ROUTE_GET_SERVERS)]
+        return [CBWParser().parse(CBWServer, server) for server in self._request(
+            "GET",
+            self._build_route(ROUTE_SERVERS))]
 
     def server(self, server_id):
         """GET request to /api/v2/servers to get all informations about a specific server"""
-        return CBWParser().parse(CBWServer, self._request(ROUTE_GET_SERVERS, [server_id]))
+        return CBWParser().parse(CBWServer, self._request(
```

Comment:
Why 3 lines ?

Reply:
Pylint doesn't want to let one line, too many characters ... (Pylint message: "Line too long 116/100")